<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Default Values</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>NB: Hit ENTER to validate changes in each cell!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="lockButton">
      <property name="text">
       <string>UNLOCK !</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lockButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>edit_button()</slot>
  <slot>reset_button()</slot>
 </slots>
</ui>
